lib/inventory_refresh/inventory_collection.rb

```diff
@@ -232,13 +227,13 @@ def unique_index_keys
 
     # @return [Array<ActiveRecord::ConnectionAdapters::IndexDefinition>] array of all unique indexes known to model
     def unique_indexes
-      @unique_indexes_cache if @unique_indexes_cache
+      return @unique_indexes_cache if @unique_indexes_cache
```

Comment:
I'm wonder if this should be split out into its own PR (along with the other cache thing), because it is an actual change in how the code works.  I'm concerned we might need some extra specs around it, but don't want to muddy up a simpler rubocop PR.